tests/unistd/pathconf.out

```diff
@@ -49,13 +49,13 @@ errno: 0
 _PC_SOCK_MAXBUF: -1
 errno: 0
 
-_PC_REC_INCR_XFER_SIZE: -1
+_PC_REC_INCR_XFER_SIZE: 4096
 errno: 0
 
-_PC_REC_MAX_XFER_SIZE: -1
+_PC_REC_MAX_XFER_SIZE: 4096
 errno: 0
 
-_PC_SYMLINK_MAX: -1
+_PC_SYMLINK_MAX: 255
```

Comment:
is it not worth matching the old values?

Reply:
I kind of decided for some values is wasn't worth it, better to be in line with musl... but maybe its better to keep this change clean.. I'll do that.

Reply:
I don't feel strongly either way. But it does feel safer to change less.

(If you prefer to change some values, that's fine too, but let's document that in the changelog then.)